packages/wallets/fractal/package.json

```diff
@@ -33,11 +33,11 @@
         "@solana/web3.js": "^1.58.0"
     },
     "dependencies": {
-        "@fractalwagmi/solana-wallet-adapter": "^0.0.8",
+        "@fractalwagmi/solana-wallet-adapter": "0.0.9",
```

Comment:
@obber should this be released as `0.1.0` so that it can be updated according to semver without changes to the adapter?